install.sh

```diff
@@ -205,6 +224,78 @@ check_for_running_collectd(){
     fi
 }
 
+check_time_ntp() {
+    if which ntpdate > /dev/null; then
+        local output="$(ntpdate -q -t 2 pool.ntp.org 2> /dev/null)"
+        if [ $? -eq 0 ]; then
+            # Parse output to see what the drift is. We have to do this in awk
+            # because bash doesn't natively handle floating point.
+            if echo "$output" | awk -v max_drift=$time_sync_max_drift "{
+                if (!/step time server .+ offset/ || NF != 11)
+                    next
+
+                if ($$10 > max_drift || $$10 < -max_drift)
+                    exit 1
+                else
+                    exit 0
+            }"; then
+                echo "NTP time in sync."
```

Comment:
might not want to state this since it isn't really in sync but within a 10 second range. do you really need to echo anything?